CRITICAL - 2022-02-28 01:38:05 --> Use of undefined constant userid - assumed 'userid' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 01:38:28 --> Use of undefined constant userid - assumed 'userid' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 01:59:06 --> Use of undefined constant user - assumed 'user' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 02:01:20 --> Use of undefined constant user - assumed 'user' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 02:01:21 --> Use of undefined constant user - assumed 'user' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 02:04:39 --> Use of undefined constant userid - assumed 'userid' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 02:20:14 --> Use of undefined constant user - assumed 'user' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 02:20:20 --> Use of undefined constant user - assumed 'user' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 02:37:49 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\ispy_erp\system\Security\Security.php(289): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\ispy_erp\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\ispy_erp\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('/user/update/[o...', 'before')
#4 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2022-02-28 04:03:07 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 04:03:07 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('info@i-spy.com', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'info@i-spy.com', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:03:15 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:03:15 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('info@i-spy.com', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'info@i-spy.com', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:04:40 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:04:40 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('info@i-spy.com', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'info@i-spy.com', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:05:16 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:05:16 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('johndoe@gmail.c...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'johndoe@gmail.c...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:08:31 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:08:31 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:11:04 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:11:04 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:11:27 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:11:27 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-02-28 05:13:04 --> Return value of CodeIgniter\BaseModel::transformDataToArray() must be of the type array, string returned
#0 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(704): CodeIgniter\BaseModel->transformDataToArray('\r\n<!DOCTYPE htm...', 'insert')
#1 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert('\r\n<!DOCTYPE htm...', false)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert('\r\n<!DOCTYPE htm...', false)
#3 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save('\r\n<!DOCTYPE htm...')
#4 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2022-02-28 05:15:46 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:15:46 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-02-28 05:22:54 --> syntax error, unexpected '?>'
#0 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include()
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(226): CodeIgniter\View\View->render('admin_template/...', Array, true)
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(62): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-28 05:23:47 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:23:47 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:24:23 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:24:23 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:24:48 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:24:48 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:24:49 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:24:49 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:26:41 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:26:41 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:29:13 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:29:13 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('info@i-spy.com', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'info@i-spy.com', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:33:32 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:33:32 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(120): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-02-28 05:34:19 --> There is no data to update.
#0 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(NULL, 'update')
#2 C:\xampp\htdocs\ispy_erp\system\Model.php(651): CodeIgniter\BaseModel->update(Array, NULL)
#3 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(157): CodeIgniter\Model->update(Array)
#4 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->update()
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-28 05:35:00 --> There is no data to update.
#0 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(NULL, 'update')
#2 C:\xampp\htdocs\ispy_erp\system\Model.php(651): CodeIgniter\BaseModel->update(Array, NULL)
#3 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(157): CodeIgniter\Model->update(Array)
#4 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->update()
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2022-02-28 05:38:44 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:38:44 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(120): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:41:38 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:41:38 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:46:04 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:46:04 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:47:59 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:47:59 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:49:55 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:49:55 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:53:40 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:53:40 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:55:46 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:55:46 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 05:55:46 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 05:55:46 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-02-28 05:56:51 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 05:57:14 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 05:57:15 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 05:57:15 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 05:57:16 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 05:57:16 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 05:57:17 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 05:57:27 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 05:58:17 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users\users.php(125): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 125, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(129): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:01:19 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(77): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:01:24 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(77): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:01:30 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(77): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:01:44 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(77): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:03:03 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(75): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:03:07 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(75): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:04:20 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(75): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:04:21 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(75): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:04:21 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(75): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-28 06:04:32 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\clients.php(149): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 149, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\ClientController.php(75): view('admin_template/...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\ClientController->insertclient()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ClientController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-28 06:13:05 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 06:13:05 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('johndoe@gmail.c...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'johndoe@gmail.c...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 06:26:41 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 06:26:41 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(118): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 06:28:40 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 06:28:40 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 06:30:56 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 06:30:56 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 06:31:39 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 06:31:39 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('snmurira@gmail....', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'snmurira@gmail....', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(116): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 06:37:51 --> Cannot add or update a child row: a foreign key constraint fails (`ispy`.`installation`, CONSTRAINT `installation_ibfk_1` FOREIGN KEY (`userid`) REFERENCES `users` (`userid`) ON DELETE NO ACTION ON UPDATE CASCADE)
ERROR - 2022-02-28 07:24:59 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 07:24:59 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(119): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-28 09:32:37 --> Table 'ispy.user' doesn't exist
CRITICAL - 2022-02-28 09:32:37 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('support@ispy.co...', 'email', Array, NULL)
#1 C:\xampp\htdocs\ispy_erp\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'support@ispy.co...', Array, Array)
#2 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ispy_erp\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ispy_erp\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(119): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#9 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
